### PR TITLE
Allow sandbox to open an URL in the current browser context

### PR DESCRIPTION
**Pull request by leoWorkingGood**
**What problem is this solving?**

- Allow sandbox to open an URL in the same tab

**How should this be manually tested?**

- Having an anchor tag with the target="_parent" attribute and it opens in the same tab

### CODE REVIEW

**Comment by victorhmp** [comment]
Hi @leoWorkingGood !
Could you provide a workspace where this feature is being used?

**Comment by leoWorkingGood** [comment]
Do I have to send the workspace, or is it already approved and it's gonna be merged?

**Comment by iaronaraujo** [comment]
Hi @leoWorkingGood ! Please provide the workspace so we can test it better!

**Comment by Klynger** [comment]
> Do I have to send the workspace, or is it already approved and it's gonna be merged?

Still send the workspace pls

**Comment by leoWorkingGood** [comment]
Here it is the workspace https://leo--worldwidegolf.myvtex.com/titleist-loyalty-rewarded

You can test with the production link and see the difference https://www.worldwidegolfshops.com/titleist-loyalty-rewarded

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Fixed
+- Allow sandbox's to manipulate its parent frame
```
Comment (victorhmp) [inline CHANGELOG.md:11]:
```suggestion
### Added
- [`allow-top-navigation`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe) to `sandbox` attribute.
```

**Comment by victorhmp** [comment]
Everything looks good @leoWorkingGood ! Thanks for your contribution!
We will merge and release this on Monday :)

**Comment by leoWorkingGood** [comment]
Okay, thank you very much!

**Pull request merged**